bin/agd

```diff
@@ -15,41 +15,8 @@
 set -ueo pipefail
 
 function fatal() {
-  echo ${1+"$@"} 1>&2
-  exit 1
-}
-
-# Use sha1sum where available or as an ultimate fallback for good error messages,
-# but otherwise use other available fallbacks.
-for cmd in sha1sum shasum sha1sum; do
-  DIGEST_CMD=$cmd
-  command -v $DIGEST_CMD &>/dev/null && break
-done
-
-# diffsha1 stamp_name find_args...
-# executes find(1) with the given arguments and saves the resulting file list,
-# then runs $DIGEST_CMD on the file list, saves the digest to file stamp_name.new,
-# and compares to the saved digest in the file stamp_name.
-# Returns success (0) if it cannot write to the temporary files, if the find results are empty,
-# or if the computed digest is the same as the saved digest.
-# Returns failure (1) if the computed digest is different.
-function diffsha1() {
-  stamp=$1
-  shift
-  find ${1+"$@"} > "$stamp.files.$$"
-  if test ! -s "$stamp.files.$$"; then
-    # echo "No new dependencies found for $stamp" 1>&2
-    rm -f "$stamp.files.$$"
-    return 0
-  fi
-  xargs $DIGEST_CMD <"$stamp.files.$$" | sort +1 > "$stamp.new" || true
-  rm -f "$stamp.files.$$"
-  if test ! -s "$stamp.new"; then
-    rm -f "$stamp.new"
-    return 0
-  fi
-  diff -u "$stamp" "$stamp.new" || return 1
-  return 0
+    echo ${1+"$@"} 1>&2
```

Comment:
What does this do?

Reply:
It echoes the `fatal` arguments to stderr with proper escaping.